core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -190,4 +196,25 @@ Command workerHeartbeat(long workerId, Map<String, Long> usedBytesOnTiers,
    * @param blockIds the ids of the lost blocks
    */
   void reportLostBlocks(List<Long> blockIds);
+
+  /**
+   * Saves the callback function to use when lost workers become alive.
+   *
+   * @param function the function to save to corresponding callback list
+   */
+  void workerAddConfCallback(Consumer<Long> function);
```

Comment:
A common naming pattern is "register{ACTION}Listener", where {ACTION} is the action that will trigger the callback. For example, `registerLostWorkerFoundListener`

Reply:
Changed to : 
```
registerLostWorkerFoundListener
 registerWorkerLostListener
registerNewWorkerConfListener
```
Thanks for the callback conventions.